integration-tests/src/test/groovy/com/okta/sdk/tests/it/ApplicationsIT.groovy

```diff
@@ -479,6 +479,9 @@ class ApplicationsIT extends ITSupport {
         client.createApplication(app)
         registerForCleanup(app)
 
+        // fix OKTA-279039
```

Comment:
nit: Do we want to explain why this sleep is needed?

Reply:
added a note.